configure.ac

```diff
@@ -49,6 +49,14 @@ AC_ARG_VAR([OMR_CROSS_CONFIGURE],
 	Compiler and feature detection tests will be invalid. All options must be configured manually.
 	One of: yes,no. (Default: no)])
 
+AC_ARG_VAR([OMR_PRODUCT_DESCRIPTION], [The description of product integerating OMR. (Default: OMR 1.0)])
```

Comment:
I would prefer a default version number of 0.1 as it may be awhile before we get to a 1.0 release.